sky/resources.py

```diff
@@ -161,6 +166,11 @@ def __repr__(self) -> str:
             else:
                 image_id = f', image_id={self.image_id!r}'
 
+        disk_type = ''
+        if self.disk_type is not None:
+            disk_desc = self.cloud.get_disk_desc(self.disk_type)
```

Comment:
We should hide the actual lower-level descriptions from the users for a better user experience.